opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java

```diff
@@ -463,7 +463,7 @@ protected void doInTransactionWithoutResult(TransactionStatus arg0) {
             node.withTreeLock(r);
 
         } catch (final Throwable e) {
-            LOG.error("Unable to schedule service {}/{}/{}", nodeId, normalizedAddress, svcName);
+            LOG.error("Unable to schedule service {}/{}/{}", nodeId, normalizedAddress, svcName, e);
```

Comment:
```suggestion
            LOG.error("Unable to schedule service {}/{}/{}. Exception message: {}", nodeId, normalizedAddress, svcName, e.getMessage());
```

Reply:
We need to look here if the Exception toString() or the getMessage() method gives useful information.

Reply:
Convention in our code is to log whole call stack, so the change looks fine.

Reply:
I would favor the original one. No need to explicitly log exception message.

Reply:
Sounds good to me. Signed the CLA.
Not sure if a tiny change like this requires an JIRA.